pytensor/configdefaults.py

```diff
@@ -423,6 +423,13 @@ def add_compile_configvars():
         in_c_key=False,
     )
 
+    config.add(
+        "optimizer_verbose_ignore",
+        "Ignore rewrites from optimizers with these names. Separate names with ','",
```

Comment:
I suggest the following, to be clear that this is about ignoring the display and not the rewrites themselves.

```suggestion
        "Don't print rewrites from optimizers with these names. Separate names with ','",
```